sdk/core/azure-core-test/pom.xml

```diff
@@ -59,6 +59,12 @@
       <version>5.4.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-engine;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.4.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-params;external_dependency} -->
```

Comment:
There is more latest version `5.5.2`  , should we use that ?
https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-params/5.5.2

Reply:
I believe there is an issue with using JUnit 5.5.* which is why we are using 5.4.2. I'm going to use 5.4.2 here since that is what we have in the external dependencies version file, if you think we should update it file an issue for discussion.